.github/CONTRIBUTING.md

```diff
@@ -28,6 +28,22 @@ IGListExperimentEnabled(self.experiments, IGListExperimentMyAwesomeChange) {
 
 Once your experiment is confirmed we will remove the option and wrapping check!
 
+## How we do major and minor releases
+
+Keep in mind:
+
+2x PRs merge into stable
+3x PRs merge into master
```

Comment:
this should be something like:

"Current release pull requests merge into `stable`"

"Next major release pull requests merge into `master`"

-------

then, we should probably give an example, something like:

if current release is 2.1.0, then any commits for 2.2.0 go into `stable` while commits for `3.0` would go to master